city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -0,0 +1,178 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.relativedelta import relativedelta
+
+
+class CookJusticeAdvisorySpider(CityScrapersSpider):
+    name = "cook_justice_advisory"
+    agency = "Cook County Justice Advisory"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.cookcountyil.gov"]
+    agenda_map = defaultdict(list)
+
+    def start_requests(self):
+        url = "https://www.cookcountyil.gov/service/justice-advisory-council-meetings"
+        yield scrapy.Request(
+            url=url, callback=self._parse_links, dont_filter=True,
+        )
+        today = datetime.now()
+        for month_delta in range(-3, 6):
+            mo_str = (today + relativedelta(months=month_delta)).strftime("%Y-%m")
+            url = (
+                "https://www.cookcountyil.gov/"
+                "calendar-node-field-date/month/{}".format(mo_str)
+            )
+            yield scrapy.Request(url=url, method="GET", callback=self.parse)
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for url in self._get_event_urls(response):
+            yield scrapy.Request(url, callback=self._parse_event, dont_filter=True)
+
+    def _parse_event(self, response):
+        """Parse the event page."""
+        agenda_map = self._get_agenda()
+        title = self._parse_title(response)
+        start = self._parse_start(response)
+        meeting_year = int(start.strftime("%y"))
```

Comment:
Instead of making two calls to `strftime` here, it might be quicker to just call once with a matching format like this

```python
links_key = start.strftime("%y-%m")
```

but that might require changing the implementation in `_parse_links`. Will comment there as well